<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>922</width>
    <height>615</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BSA-analytics</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>4</number>
    </property>
    <property name="topMargin">
     <number>4</number>
    </property>
    <property name="rightMargin">
     <number>4</number>
    </property>
    <property name="bottomMargin">
     <number>4</number>
    </property>
    <item>
     <widget class="QScrollArea" name="frame">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>910</width>
         <height>552</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <pointsize>48</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Waiting for data input...</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>922</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionOpen_Binary"/>
    <addaction name="actionCustom_open"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionPrint"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionAutoDraw"/>
    <addaction name="actionAxes"/>
    <addaction name="actionNet"/>
    <addaction name="actionLive"/>
    <addaction name="actionFast"/>
    <addaction name="actionClone"/>
    <addaction name="separator"/>
    <addaction name="actionSound_mode"/>
   </widget>
   <widget class="QMenu" name="menuPulsars">
    <property name="title">
     <string>&amp;Pulsars</string>
    </property>
    <addaction name="actionPulsar_searcher"/>
    <addaction name="separator"/>
    <addaction name="actionPulsar_analytics"/>
    <addaction name="actionPulsar_analytics_low_memory"/>
    <addaction name="separator"/>
    <addaction name="actionPrecise_search"/>
    <addaction name="actionPrecise_timing"/>
    <addaction name="separator"/>
    <addaction name="actionFlow_detecter"/>
    <addaction name="actionRotation_measure"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuPulsars"/>
   <addaction name="menuEdit"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="text">
    <string>&amp;Open</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionAutoDraw">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;AutoDraw</string>
   </property>
  </action>
  <action name="actionAxes">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>A&amp;xes</string>
   </property>
  </action>
  <action name="actionNet">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Net</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About &amp;Qt</string>
   </property>
  </action>
  <action name="actionCustom_open">
   <property name="text">
    <string>&amp;Custom open</string>
   </property>
  </action>
  <action name="actionOpen_Binary">
   <property name="text">
    <string>Open &amp;Binary</string>
   </property>
  </action>
  <action name="actionFast">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Fast</string>
   </property>
  </action>
  <action name="actionClone">
   <property name="text">
    <string>&amp;Clone</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionLive">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Live</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="text">
    <string>P&amp;rint</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionPulsar_searcher">
   <property name="text">
    <string>&amp;Pulsar searcher</string>
   </property>
  </action>
  <action name="actionPulsar_analytics">
   <property name="text">
    <string>Pulsar &amp;analytics</string>
   </property>
  </action>
  <action name="actionPrecise_search">
   <property name="text">
    <string>Precise &amp;search</string>
   </property>
  </action>
  <action name="actionSound_mode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Sound mode</string>
   </property>
  </action>
  <action name="actionPulsar_analytics_low_memory">
   <property name="text">
    <string>Pulsar analytics &amp;low memory</string>
   </property>
  </action>
  <action name="actionPrecise_timing">
   <property name="text">
    <string>Precise &amp;timing</string>
   </property>
  </action>
  <action name="actionFlow_detecter">
   <property name="text">
    <string>&amp;Flow detecter</string>
   </property>
  </action>
  <action name="actionRotation_measure">
   <property name="text">
    <string>Rotation measure</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
